variables.sh

```diff
@@ -12,6 +12,9 @@ DUMP=data/dumps/${lang}
 MOSES=$HOME/IdeaProjects/mosesdecoder
 SUBWORDNMT=$HOME/IdeaProjects/subword-nmt
 
+# Location of the shared task data.
+SHARED_TASK_DATA=$HOME/staple2020/staple-2020-train
```

Comment:
Is this used? It looks duplicated against a variable in `preprocess.sh`

Reply:
Ah, thanks. It shouldn't be in `preprocess.sh`